Bookish/Controllers/CatalogController.cs

```diff
@@ -114,6 +114,18 @@ public IActionResult BookList()
         }
         return View(bvm);
     }
+    
+    public IActionResult CopyList(int inputId)

```

Comment:
I'd call this method something like `GetCopies` and give it an `[HttpGet]` annotation

Reply:
I'd also change inputId to e.g. inputBookId, it's not immediately obvious what it is